ui/desktop/app/templates/scopes/scope/projects/sessions/index.hbs

```diff
@@ -115,37 +115,53 @@
     </Rose::Toolbar>
 
     {{#if this.sorted}}
-      <Rose::Table as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
-            <row.headerCell>{{t 'resources.target.title'}}</row.headerCell>
-            <row.headerCell>{{t 'form.proxy.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.started.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.status.label'}}</row.headerCell>
-            <row.headerCell />
-          </header.row>
-        </table.header>
-        <table.body as |body|>
-          {{#each this.sorted as |model|}}
-            <body.row as |row|>
-              <row.headerCell class='nowrap'>
-                <Rose::Icon
-                  class={{concat 'session-status-' model.status}}
-                  @name='flight-icons/svg/entry-point-16'
-                  @size='medium'
-                />
+      <Hds::Table
+        @model={{this.sorted}}
+        @columns={{array
+          (hash label=(t 'form.id.label'))
+          (hash label=(t 'resources.target.title'))
+          (hash label=(t 'form.proxy.label'))
+          (hash label=(t 'form.started.label'))
+          (hash label=(t 'form.status.label'))
+          (hash label='')
+        }}
+        @valign='middle'
+      >
+        <:body as |B|>
+          <B.Tr>
+            <B.Td>
+              {{#if (can 'read model' B.data)}}
+                {{#if B.data.isActive}}
+                  <Hds::Badge
+                    @text={{t 'resources.session.title'}}
+                    @icon='entry-point'
+                    @isIconOnly={{true}}
+                    @color='success'
+                    {{! there are plans to remove this class. It is here to make the test 
+                    pass. This is going to change later so Cameron suggested it's ok to
+                    have it for now.}}
```

Comment:
```suggestion
                    {{! TODO: Remove this class once the table gets refactored. 
                    It's not currently needed but is only used for tests to pass. }}
```

Small suggestion to actually use a `TODO` (which makes it easier to find these comments)